src/AppInstallerRepositoryCore/Rest/Schema/1_0/Json/ManifestDeserializer.cpp

```diff
@@ -428,18 +428,32 @@ namespace AppInstaller::Repository::Rest::Schema::V1_0::Json
         return installer;
     }
 
-    std::optional<Manifest::Dependency> ManifestDeserializer::DeserializeDependency(const web::json::value& dependenciesObject) const
+    std::optional<Manifest::DependencyList> ManifestDeserializer::DeserializeDependency(const web::json::value& dependenciesObject) const
     {
         if (dependenciesObject.is_null())
         {
             return {};
         }
 
-        Manifest::Dependency dependency;
+        Manifest::DependencyList dependencyList;
 
-        dependency.WindowsFeatures = ConvertToManifestStringArray(JsonHelper::GetRawStringArrayFromJsonNode(dependenciesObject, JsonHelper::GetUtilityString(WindowsFeatures)));
-        dependency.WindowsLibraries = ConvertToManifestStringArray(JsonHelper::GetRawStringArrayFromJsonNode(dependenciesObject, JsonHelper::GetUtilityString(WindowsLibraries)));
-        dependency.ExternalDependencies = ConvertToManifestStringArray(JsonHelper::GetRawStringArrayFromJsonNode(dependenciesObject, JsonHelper::GetUtilityString(ExternalDependencies)));
+        const auto& wfIds = ConvertToManifestStringArray(JsonHelper::GetRawStringArrayFromJsonNode(dependenciesObject, JsonHelper::GetUtilityString(WindowsFeatures)));
+        for (auto id : wfIds)

```

Comment:
The temporary `wfIds` should not be a temporary, and this should then use `auto&&` and below should use `std::move`.